pages/common/st-util.md

```diff
@@ -0,0 +1,15 @@
+# st-util
+
+> Run GDB (GNU Debugger) server to interact with STM32 ARM Cortex microcontoller.
+
+- Run GDB server on port 4500:
+
+`st-util --listen_port {{4500}}`
```

Comment:
Lets shift it back to `-p` as suggested by @waldyrious. Sorry for the back and forth.